### PR TITLE
Add compat data for SVGStopElement

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/SVGStopElement

### CODE REVIEW

File: api/SVGStopElement.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGStopElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStopElement",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
```
Comment (ExE-Boss) [inline api/SVGStopElement.json]: Please sort the `support` block alphabetically (see issue #398 and PR #1882), this applies to all your other PRs.

File: api/SVGStopElement.json [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SVGStopElement.json]: All 3 flavors of Chrome should be `true`.

https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/svg/svg_stop_element.idl

File: api/SVGStopElement.json [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/SVGStopElement.json]: Both Operas should be `true`.

**Comment by maboa** [comment]
That should be all issues addressed :)

**Pull request merged**